easybuild/easyblocks/f/ferret.py

```diff
@@ -48,80 +53,156 @@ def configure_step(self):
         """Configure Ferret build."""
 
         buildtype = "x86_64-linux"
-
-        try:
-            os.chdir('FERRET')
-        except OSError, err:
-            raise EasyBuildError("Failed to change to FERRET dir: %s", err)
+        if LooseVersion(self.version) < LooseVersion("7.3"):
+            try:
+                os.chdir('FERRET')
```

Comment:
please replace this `try`/`except` block by using the `change_dir` function from `filetools`